### PR TITLE
mx-assets readme

### PR DESCRIPTION
**Pull request by alyn509**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,14 @@ Here’s a prefilled template for the .json file to get you started:
 
 ```JSON
 {
-    "website": "https://www.elrondtoken.com",
-    "description": "The ERD token is the utility token of Elrond Token",
+    "website": "https://www.multiversxtoken.com",
+    "description": "The ERD token is the utility token of MultiversX Token",
```
Comment (SebastianMarian) [inline README.md:19]: EGLD instead ERD?

File: README.md [hunk 2]
```diff
@@ -15,14 +15,14 @@ Here’s a prefilled template for the .json file to get you started:
...
     "ledgerSignature": "1234",
     "social": {
-        "email": "erd-token@elrond.com",
-        "blog": "https://www.elrondtoken.com/ERD-token-blog",
+        "email": "erd-token@multiversx.com",
```
Comment (SebastianMarian) [inline README.md:22]: erd here in the whole file?

File: README.md (hunk 1 above)
Comment (alyn509) [reply README.md:19]: as I heard all these are fictive data

File: README.md (hunk 2 above)
Comment (alyn509) [reply README.md:22]: that's how the template was

**Review by LucianMincu — APPROVED** [review]

**Pull request merged**